docs/joss_paper/paper.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 'A Python astronomy package for HiPS: Hierarchical Progressive Surveys'
+tags:
+- python
+- astronomy
+- science
+- image-viewer
+- image-processing
+
+authors:
+- name: Christoph Deil
+  orcid:
+  affiliation: 1
+
+- name: Thomas Boch
+  orcid:
+  affiliation: 2
+
+- name: Adeel Ahmad
+  orcid: 0000-0001-5035-4497
+  affiliation: 3
+
+affiliations:
+- name: Max Planck Institute for Nuclear Physics, Heidelberg, Germany
+  index: 1
+- name: Centre de Données astronomiques de Strasbourg, Strasbourg, France
+  index: 2
+- name: National University of Computer and Emerging Sciences, Islamabad, Pakistan
+  index: 3
+  
+date: 5 July 2018
+bibliography: paper.bib
+---
+
+# Summary
+HiPS (Hierarchical Progressive Surveys) [@2015A&A...578A.114F] is built upon the HEALPix framework and provides a way to store large astronomical survey sky images and catalog datasets on servers (such as HiPS at CDS  (\url{http://aladin.u-strasbg.fr/hips}), that allows clients to efficiently fetch only the image tiles or catalog parts for a given region of the sky they are interested in.
```

Comment:
I would suggest to add this reference when mentioning HEALPix for the first time:
http://adsabs.harvard.edu/abs/2005ApJ...622..759G
Of course it's also referenced form the HiPS paper, but given that we use HEALPix a lot and it's the basis of how this works, I think it's good to have the explicit reference in this paper.